lib/views/after_auth_screens/events/event_calendar.dart

```diff
@@ -130,12 +130,10 @@ _AppointmentDataSource _getCalendarDataSource(List<Event> eventsList) {
   eventsList.forEach((event) {
     final startDate = DateFormat('yMd').parse(event.startDate!);
     print("${event.startTime!}##############################");
-    final startTime =
-        DateFormat('Hms', 'en_US').parse(event.startTime ?? '14:23:01');
+    final startTime = _parseTime(event.startTime ?? '14:23:01');
     // .parse(event.startTime!);
     final endDate = DateFormat('yMd').parse(event.endDate!);
-    final endTime =
-        DateFormat('Hms', 'en_US').parse(event.endTime ?? '14:23:01');
+    final endTime = _parseTime(event.endTime ?? '21:23:01');
```

Comment:
Changes looks fine but why we are using 14.23.01 as a default ?

Reply:
It was already there as default value, therefore I just used that